.gitlab-ci.yml

```diff
@@ -109,7 +109,7 @@ publish:image:
 publish:disclaimer:
   stage: publish
   services:
-    - docker:dind
+    - docker:19.03.5-dind
```

Comment:
For the publish jobs it is not necessary. Just FYI, it is a good practice anyway ;)

Reply:
Oh right since they are basically just pushing a file instead of any virtualisation - thanks for the learning!